### DISCUSSION TITLE
Multiple Blogs

### DISCUSSION THREAD

**Discussion by rjkreider** [jlelse/GoBlog#56] [Q&A] [answered]
I am tinkering with the idea of multiple blogs.

In the configuration, I specified another blog.

I can access the second blog homepage without issue and see the posts with excerpts.  I can also create posts in the second blog.

When I try to access a post on the second blog, the site hangs.  I’m not getting much back from GoBlog that is running on the terminal (non docker); just a select statement.

If I’m logged in, I can access the second blog and access the second blog’s full posts.

Thoughts where to look?

ETA: I just noticed this is the same behavior on my main blog.  If I’m not logged in, the site hangs when I try to view a full post on the main (default) blog.

**Comment by jlelse** [comment, marked as answer]
Did you pull the latest updates? On Friday I pushed a faulty change that I corrected later.

**Reply by rjkreider** [reply]
I pulled yesterday and everything is good now, thanks!
